src/lib/transaction_snark/test/util.ml

```diff
@@ -71,19 +72,22 @@ let apply_zkapp_command ledger zkapp_command =
         [ ( `Pending_coinbase_init_stack init_stack
           , `Pending_coinbase_of_statement
               (pending_coinbase_state_stack
-                 ~state_body_hash:genesis_state_body_hash )
+                 ~state_body_hash:genesis_state_body_hash
+                 ~global_slot:Mina_numbers.Global_slot.zero )
```

Comment:
`.(succ zero)`? No transaction will ever have slot `zero`, because the genesis block has no transactions ;)